pymysqlreplication/event.py

```diff
@@ -57,7 +58,14 @@ def _verify_event(self):
         data = self.packet.read(19 + self.event_size)
         footer = self.packet.read(4)
         byte_data = zlib.crc32(data).to_bytes(4, byteorder="little")
-        self._is_event_valid = True if byte_data == footer else False
+        if byte_data == footer:
+            self._is_event_valid = True
+        else:
+            self._is_event_valid = False
+            logging.error(
+                f"An CRC32 has failed for the event type {self.event_type}, "
+                "indicating a potential integrity issue with the data."
+            )
```

Comment:
```suggestion
      self._is_event_valid = True if byte_data == footer else False
      if not self._is_event_valid:
            logging.error(
                f"An CRC32 has failed for the event type {self.event_type}, "
                "indicating a potential integrity issue with the data."
            )
```
@why-arong 
I think this looks more readable, what do you think?
I think the code you wrote is good, too.

Reply:
I think that code reads better!

How about error message? Is there any additional information we can provide besides `'self.event_type'`??

Reply:
I think that's enough